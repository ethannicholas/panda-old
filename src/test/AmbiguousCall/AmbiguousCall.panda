class AmbiguousCall {
    @class
    method ambiguous(i:Int, r:Real) {
    }

    @class
    method ambiguous(r:Real, i:Int) {
    }
}

AmbiguousCall.ambiguous(0, 0)