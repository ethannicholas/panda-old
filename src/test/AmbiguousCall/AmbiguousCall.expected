error: AmbiguousCall.panda:11:1: call to 'ambiguous' with parameters (panda.core.Int8, panda.core.Int8) is ambiguous. The following methods are all equally good matches:
    method ambiguous(i:panda.core.Int32, r:panda.core.Real64) (AmbiguousCall.panda:3:5)
    method ambiguous(r:panda.core.Real64, i:panda.core.Int32) (AmbiguousCall.panda:7:5)
1 error
